client/util/format-date.test.ts

```diff
@@ -0,0 +1,77 @@
+import { formatDistance, formatToLocaleString, defaultEnvironment, Environment } from './format-date'
+import { subDays } from 'date-fns'
+import { ja, enUS } from 'date-fns/locale'
```

Comment:
This reduces bundle size. But on the other hand, it drops many locales support.

I tried to resolve this problem by dynamic import with webpack. Crowi serves JS stuffs with /static path, however webpack’s dynamic loader loads splitted JS stuff with root path.

I think we can put off this problem. If we found correct webpack configuration in future, I will resolve this issue.